astropy/coordinates/builtin_frames/galactocentric.py

```diff
@@ -216,6 +276,12 @@ def get_roll0(cls):
         # API, so it's better for it to be accessable from Galactocentric
         return _ROLL0
 
+    def get_references(self):
+        if self._references is None:
+            raise ValueError("No references were provided or are available for "
```

Comment:
I think I'd make this a property `references` (or perhaps `frame_references`) instead, and then here raise `AttributeError`.

Reply:
I did this, but I guess if we make it an attribute like that, we don't really need a `get_` method?